File: trade-service/src/main/java/finos/traderx/tradeservice/model/TradeOrder.java
package finos.traderx.tradeservice.model;

public class TradeOrder {

    public String id;
    private String state;
    private String security;
    private Integer quantity;
    private Integer accountId;
    private TradeSide side;
    private String action;

    public TradeOrder(){}
    
    public TradeOrder(String id, int accountId, String security, TradeSide side, int quantity, String action) {
        this.accountId = accountId;
        this.security = security;
        this.side = side;
        this.quantity = quantity;
        this.id = id;
        this.action = action;
    }

    public String getId() {
        return id;
    }

    public String getAction() {
        return action;
    }

    public String getState() {
        return state;
    }

    public Integer getAccountId() {
        return accountId;
    }

    public String getSecurity() {
        return security;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public TradeSide getSide() {
        return side;
    }
}

File: morphir/triples/traderx/models/TradeOrder.scala
package traderx.models

/** Generated based on Models.TradeOrder
*/
object TradeOrder{

  final case class TradeOrder(
    id: morphir.sdk.String.String,
    state: morphir.sdk.String.String,
    security: morphir.sdk.String.String,
    quantity: morphir.sdk.Basics.Int,
    accountId: morphir.sdk.Basics.Int,
    side: traderx.models.TradeSide.TradeSide
  ){}

}